source/Octopus.Tentacle/Kubernetes/Scripts/RunningKubernetesJob.cs

```diff
@@ -75,6 +78,8 @@ public class RunningKubernetesJob : IRunningScript
 
             outputStreamWriter = new KubernetesJobOutputStreamWriter(workspace);
 
+            State = ProcessState.Pending;
```

Comment:
We weren't setting the initial state to be `Pending`, so it isn't immediately completed... which was wrong